querydsl-core/src/main/java/com/querydsl/core/types/QBean.java

```diff
@@ -73,6 +78,10 @@
         return rv.build();
     }
 
+    private static boolean isAliasSplitted(Expression<?> expr) {
```

Comment:
Sorry. I forgot to respond to your question. I would rename the method. `isCompoundExpression`?